packages/experiments-realm/blog-app.gts

```diff
@@ -337,10 +339,26 @@ class BlogAppTemplate extends Component<typeof BlogApp> {
         name: summary.id.substring(lastIndex + 1),
       };
       filter.cardRef = cardRef;
+
+      let realmUrl = this.args.model[realmURL];
+      if (!this.args.model.id || !realmUrl?.href) {
+        throw new Error(`Missing card id or realm url`);
+      }
+      let relativeTo = relativeURL(
+        new URL(this.args.model.id),
+        new URL(`${cardRef.module}/${cardRef.name}`),
+        realmUrl,
+      );
+      if (!relativeTo) {
+        throw new Error(`Missing relative url`);
+      }
       filter.query = {
         filter: {
           on: cardRef,
-          eq: { 'blog.id': this.args.model.id! },
+          any: [
+            { eq: { 'blog.id': this.args.model.id } },
+            { eq: { 'blog.id': relativeTo } },
+          ],
         },
```

Comment:
This is to cover both relative and absolute urls in the linksTo search.

Reply:
@habdelra @ef4 this seems like something we should normalize in our query engine